src/App.vue

```diff
@@ -41,6 +48,32 @@ const showContextMenu = (event: MouseEvent) => {
   }
 }
 
+const checkSubscriptionStatus = async () => {
+  if (!isCloud) return
+
+  if (!isLoggedIn.value) return
+
+  try {
+    const status = await authActions.fetchSubscriptionStatus()
```

Comment:
Another reason to break out the subscription work, we can do what @Myestery in https://github.com/Comfy-Org/ComfyUI_frontend/pull/6063 an dynamically load the subscription code and components only when we need them.